### DISCUSSION TITLE
[BUG] when_made label inconsistency

### DISCUSSION THREAD

**Discussion by baahar** [etsy/open-api#810] [Q&A]
Hi,

I have encountered a strange behavior. When a user sets the when_made attribute of a listing to "before 2004" via the Etsy website and saves it, the API returns "before_1700" for it. 

I went through all possible labels to make sure the others work. When the label "2000-2003" is selected, the API returns "2004_2009"

I hope this can be resolved soon. This and the return policy update issue are kind of urgent, because they were marked as breaking changes.

**Comment by nateswart** [comment]
Can you provide the following information please?
- what API endpoints or URLs were you seeing this behavior on?
- are you able to share any example public listing IDs where this is happening?

Thanks!

**Comment by baahar** [comment]
Hi,

thanks for getting back so quickly. 

- the endpoint I'm using to get the listing data is:
https://developers.etsy.com/documentation/reference/#operation/getListingsByShop

- I've just changed the when_made label on the following active listing to demonstrate the behavior: https://www.etsy.com/listing/167845824

as I said, via the endpoint above the label I'm getting is "before_1700"

**Comment by nateswart** [comment]
We did identify an issue here and have released a fix.

**Reply by baahar** [reply]
Hi,

Thanks for the fix. I can confirm that the label for "2000_2003" is returned correctly, but "before_2004" still returns "before_1700".

**Comment by baahar** [comment]
Hi again,

"before_2004" still returns "before_1700". The other labels return correctly.
